pkg/ast/spl/spl.peg

```diff
@@ -576,6 +580,33 @@ Start <- SPACE? initialSearch:(InitialSearchBlock) filterBlocks:(FilterBlock)* q
     return q, nil
 }
 
+IndexAssign <- index:("_index") EQUAL indexName:String {
+    return removeQuotes(indexName), nil
+}
+
+IndexExpression <- first:(IndexAssign) rest:(OR IndexAssign)* {
```

Comment:
nit: probably add a TODO here, since we want to allow other index selection later

Reply:
addressed in the latest commit.